src/snowflake/connector/network.py

```diff
@@ -182,6 +183,10 @@
 ID_TOKEN_AUTHENTICATOR = "ID_TOKEN"
 USR_PWD_MFA_AUTHENTICATOR = "USERNAME_PASSWORD_MFA"
 
+LOGIN_REQUEST_PATTERN = re.compile(
+    r"/session/v1/login-request\?request_id=[a-zA-Z0-9-]+&request_guid=[a-zA-Z0-9-]+"
+)
```

Comment:
@sfc-gh-mkeller do you think this is okay or should I just restrict it to a simple string match with `/session/v1/login-request`